adyenv6core/src/com/adyen/v6/factory/AdyenRequestFactory.java

```diff
@@ -149,13 +149,14 @@ public RefundRequest createRefundRequest(final String merchantAccount, final Big
         return new RefundRequest().fillAmount(String.valueOf(amount), currency.getCurrencyCode()).merchantAccount(merchantAccount).originalReference(authReference).reference(merchantReference);
     }
 
-    public DirectoryLookupRequest createListPaymentMethodsRequest(final BigDecimal amount, final String currency, final String countryCode) {
+    public DirectoryLookupRequest createListPaymentMethodsRequest(final BigDecimal amount, final String currency, final String countryCode, final String shopperLocale) {
         Amount amountData = Util.createAmount(amount, currency);
 
         return new DirectoryLookupRequest().setCountryCode(countryCode)
                                            .setMerchantReference("GetPaymentMethods")
                                            .setPaymentAmount(String.valueOf(amountData.getValue()))
-                                           .setCurrencyCode(amountData.getCurrency());
+                                           .setCurrencyCode(amountData.getCurrency())
+                                           .setShopperLocale(shopperLocale);
```

Comment:
Can't find the setShopperLocale in the adyen-java-api-library-1.2.0 version of the code.https://github.com/Adyen/adyen-java-api-library/blob/1.2.0/src/main/java/com/adyen/model/hpp/DirectoryLookupRequest.java

Reply:
It is to be included the latest develop branch https://github.com/Adyen/adyen-java-api-library/pull/62 but not yet under a tagged version.
The jar file is packaged with the latest chagnes.